runtime/src/main/java/org/corfudb/runtime/object/VersionLockedObject.java

```diff
@@ -117,6 +115,17 @@
      */
     private final Logger correctnessLogger = LoggerFactory.getLogger("correctness");
 
+    private static final String VLO_OPTIMISTIC_READ = CorfuComponent.OBJECT.toString() +
+            "vlo.optimistic-read";
+    private static final String VLO_UPDATED_OBJECT_READ = CorfuComponent.OBJECT.toString() +
+            "vlo.updated-object-read";
+    private static final String VLO_UPDATE = CorfuComponent.OBJECT.toString() + "vlo.update";
+    private static final String VLO_GC = CorfuComponent.OBJECT.toString() + "vlo.gc";
+
+    Timer optimisticReadTimer;
+    Timer updatedObjectReadTimer;
+    Timer vloUpdateTimer;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651478)